doc/HOWTO_forks.md

```diff
@@ -41,3 +41,19 @@ the commits in your branch so you only need to bring one commit.
 git fetch nu
 git cherry-pick XXXXX
 ```
+
+# Deploy
+From the root of your fork:
+
+- `bundle exec cap [staging_name] deploy`
+
+Find `staging_name` by the name of the `.rb` files in `config/deploy`
+
+To confirm deploy was successful:
+
+- `ssh nucore@nucore.rrc.uic.edu`
```

Comment:
since this looks like it's uic-specific, should we have it be more generic / say that this is an example